app/views/organizations/show.html.erb

```diff
@@ -0,0 +1,6 @@
+<h1 class="page-header">
+  <%= @organization.login %>
+</h1>
+
+<%= link_to 'Remove Classroom', @organization, class: 'btn btn-sm btn-danger', role: 'button',
```

Comment:
What does remove classroom do? Will it reset the organization? Anything we need to warn the user about?